keyboards/handwired/split89/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
stock pro micro? and not an elite c or something?
```suggestion
BOOTLOADER = caterina
```

Reply:
Thanks for this.  Yes, this is a stock pro micro unit.  Updated!  :)